package.json

```diff
@@ -61,5 +64,30 @@
   },
   "dependencies": {
     "github-contributors-list": "https://github.com/woocommerce/github-contributors-list/tarball/master"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "linters": {
+      "*.php": [
+        "php -d display_errors=1 -l",
+        "composer run-script phpcs"
+      ],
+      "*.scss": [
+        "stylelint --syntax=scss --fix",
+        "git add"
```

Comment:
Could you please explain why `git add` is necessary here and in the block below?

Reply:
`stylelint --fix` will make changes again in the files, so we need to use `git add` in order to include all in the same commit.